cloudferrylib/scheduler/scenario.py

```diff
@@ -108,6 +115,17 @@ def __init__(self, tasks, key, scenario):
         self.links = set()
         self.active_tasks = set()
 
+    @staticmethod
+    def check_structure(body):
+        if not body or type(body) is not dict:
```

Comment:
`type(body) is not dict` -> `not isinstance(body, dict)`
Probably in the future we will have a special class inherited from the dict to store the configuration and we will get the problem with this unit test